tests/input_set.py

```diff
@@ -135,6 +135,108 @@ def create_geometry(self):
         super(PinCellInput, self).create_geometry()
 
 
+class GridInput(InputSet):
+    """A simple grid problem"""
+
+    def create_materials(self):
+        """Instantiate C5G7 Materials."""
+        self.materials = \
+            openmoc.materialize.load_from_hdf5(filename='c5g7-mgxs.h5',
+                                               directory='../../sample-input/')
+
+    def create_geometry(self):
+        """Instantiate a 3x3 grid Geometry."""
+
+        xplanes = [None] * 4
```

Comment:
Add a comment above to describe what this code block does (i.e., sets up planes to form a grid)